### ISSUE TITLE
Plugin not working with Craft 5.0.5

### ISSUE THREAD

**Issue by pascalminator** [verbb/hyper#144]
### Describe the bug

After updating Craft to 5.0.5, Hyper no longer works. There is nothing showing in the Hyper fields when editing pages, only the title. I had to downgrade to Craft 5.0.4 to get it working again. 

### Steps to reproduce

1. Update Craft to 5.0.5

### Craft CMS version

5.0.5

### Plugin version

2.0.0-beta.8

### Multi-site?

No

**Comment by engram-design** [comment]
Seems to be all okay on my end! Any JS errors in the developer console? Cleared your caches?

**Comment by pascalminator** [comment]
I tried to clear all caches and it still does the same problem. I checked the console and there is no error. Weirdly enough, this problem also happens with an other Verbb plugin too: the exact same problem is happening with Icon Picker when I update to 5.0.5 but resolves when I downgrade to 5.0.4.

Here's an example with Hyper on Craft 5.0.4:
![image](https://github.com/verbb/hyper/assets/13754636/723825dd-d6b1-4df4-84d0-9124b0f18440)

And after updating to Craft 5.0.5:
![image](https://github.com/verbb/hyper/assets/13754636/7cfbe078-e741-4bea-a2e9-3df4da425d8a)
![image](https://github.com/verbb/hyper/assets/13754636/f74b775f-4289-48a0-9dc0-ded359a7d639)

**Comment by engram-design** [comment]
So sounds like something related to Vue or Vite if Icon Picker is also having an issue, as both are Vue-based fields.

I've been trying to replicate this, but no luck so far. Something unique to your install might be causing this, but still looking into it.

**Comment by pascalminator** [comment]
You're right, I just pushed the update in production and it is working fine. I don't know why this is happening in my dev environment but it looks like it is more my problem than yours at this point.

**Comment by pascalminator** [comment]
hyper-xxx.js and hyper-xxx.css are not loading at all in dev:
![image](https://github.com/verbb/hyper/assets/13754636/994a70f5-e763-4315-b49d-4b9b8b87cda1)

But they are in prod:
![image](https://github.com/verbb/hyper/assets/13754636/57f2cc2c-4b25-4bf6-bb6d-69bbee0750c3)

Something changed in Craft 5.0.5, because it is working perfectly fine in any previous version <= 5.0.4.

**Comment by engram-design** [comment]
Interesting that it's happening on your dev environment and not on production. Could almost be a race condition somewhere/somehow. Anything about your dev environment worth noting?

**Comment by pascalminator** [comment]
My dev environment is running on Wampserver 3.3.5 (yes, I know DDEV is the recommended tool but I never encountered any issues with my setup before this and I've been working with Craft since 2015) on Windows 11 with PHP 8.2, MySQL 8.3 and Apache 2.4.54.2. I tried with different versions of PHP and Apache without any luck.

Since my production environment running on CloudLinux does not have this issue and you are not able to reproduce it, I have a theory: maybe the problem is related to case sensitivity somewhere. Windows is case sensitive, but MacOS and Linux aren't.

**Comment by engram-design** [comment]
No stress, I use MAMP 😄 But nice to know you're on Windows, I'll spin up an example install just in case.

**Comment by pascalminator** [comment]
So... I found the problem with a friend. It is Craft related and not a bug from your plugin. In 5.0.5 they changed this:
![image](https://github.com/verbb/hyper/assets/13754636/3f9ddf23-d03c-4604-8e77-04c3fdb17573)

and if we go back it fixed the problem with Wamp:
![image](https://github.com/verbb/hyper/assets/13754636/9a7bf804-0737-4d99-8d84-3b590877482d)

Anyway, I finally decided to switch to DDEV a few days ago because I feel like i'll encounter more of these weird problems if I keep using Wamp.
